model/outgoing_webhook_test.go

```diff
@@ -5,142 +5,92 @@ package model
 
 import (
 	"net/url"
-	"reflect"
 	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
 func TestOutgoingWebhookJson(t *testing.T) {
 	o := OutgoingWebhook{Id: NewId()}
 	json := o.ToJson()
 	ro := OutgoingWebhookFromJson(strings.NewReader(json))
 
-	if o.Id != ro.Id {
-		t.Fatal("Ids do not match")
-	}
+	assert.Equal(t, o.Id, ro.Id, "Ids do not match")
 }
 
 func TestOutgoingWebhookIsValid(t *testing.T) {
 	o := OutgoingWebhook{}
-
-	if err := o.IsValid(); err == nil {
-		t.Fatal("should be invalid")
-	}
+	assert.NotNil(t, o.IsValid(), `empty declaration should be invalid`)
```

Comment:
It's not a big deal but to keep consistent I would try to use double quotes `"` instead of backticks `` ` `` for strings that don't need special formatting.